server/src/authorizers/agents.js

```diff
@@ -16,12 +16,35 @@ async function currentAgentIsAuthenticatedRule(root, args, context, info) {
   return result;
 }
 
+/**
+ * @param {*} root
+ * @param {object} args
+ * @param {Schema.Context} context
+ * @param {object} info
+ * @return {Promise<boolean>}
+ */
+async function currentAgentIsAdminRule(root, args, context, info) {
```

Comment:
can we call this ``currentAgentIsApplicationAdminRule`` or similar, and use similar terminology throughout this to distinguish ``applicationAdmin`` from ``channelAdmin``?

Reply:
Updated